### PR TITLE
Add edit customer command, restriction for phone number (customer and driver), update ug

### PR DESCRIPTION
**Pull request by zxgoh**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/EditCustomerCommandParser.java [hunk 1]
```diff
@@ -14,22 +14,21 @@
...
-     * and returns an EditCommand object for execution.
+     * Parses the given {@code String} of arguments in the context of the EditCustomerCommand
+     * and returns an EditCustomerCommand object for execution.
      * @throws ParseException if the user input does not conform the expected format
      */
-    public EditCommand parse(String args) throws ParseException {
+    public EditCustomerCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_TAG);
```
Comment (hieunm1821) [inline src/main/java/seedu/address/logic/parser/EditCustomerCommandParser.java:34]: can remove PREFIX_TAG

File: src/main/java/seedu/address/logic/parser/EditDishCommandParser.java [hunk 2]
```diff
@@ -11,7 +11,7 @@
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
- * Parses input arguments and creates a new EditCommand object
+ * Parses input arguments and creates a new EditCustomerCommand object
```
Comment (hieunm1821) [inline src/main/java/seedu/address/logic/parser/EditDishCommandParser.java:14]: This one should be EditDishCommand.

**Review by hieunm1821 — APPROVED** [review]
Just need to fix minor errors before merging.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -85,25 +85,36 @@ This project is based on the AddressBook-Level3 project created by the [SE-EDU i
...
 ## Customer features
 
-### Adding a customer: `add`
+### Adding a customer: `addcustomer`
 
-Adds a customer to the addressbook.
+Adds a customer to the database.
 
-Format: `add n/{name} a/{address} p/{phone}`
+Format: `addcustomer n/{name} a/{address} p/{phone}`
```
Comment (jaegarpoon) [inline docs/UserGuide.md:92]: i edited the format to UPPER_CASE in my latest PR, following all the message_usages, maybe can update this some time later!

**Review by jaegarpoon — APPROVED** [review]
Looks good, you can merge it in first, i will update the UG later on as there will be conflict! :)

File: docs/UserGuide.md (hunk 3 above)
Comment (zxgoh) [reply docs/UserGuide.md:92]: ok will change it!

**Pull request merged**